src/Aspire.Hosting.Azure.EventHubs/EventHubsEmulatorContainerImageTags.cs

```diff
@@ -1,11 +1,11 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-namespace Aspire.Hosting.Nats;
+namespace Aspire.Hosting.Azure.EventHubs;
 
 internal static class EventHubsEmulatorContainerImageTags
 {
-    public const string Registry = "messagingemulators.azurecr.io";
-    public const string Image = "microsoft/azure/eventhubs/emulator";
+    public const string Registry = "mcr.microsoft.com";
+    public const string Image = "azure-messaging/eventhubs-emulator";
     public const string Tag = "latest";
```

Comment:
Looks like there isn't a stable version number for this?

Reply:
Not yet https://mcr.microsoft.com/en-us/product/azure-messaging/eventhubs-emulator/tags

```
latest
1.2.3-preview-arm64 
1.2.3-preview-amd64 
```